ci-operator/config/ComplianceAsCode/content/ComplianceAsCode-content-master__4.15.yaml

```diff
@@ -810,6 +810,102 @@ tests:
         requests:
           cpu: 100m
     workflow: ipi-aws
+- as: e2e-aws-rosa-sts-hcp-fips-ocp4-pci-dss-node-weekly
```

Comment:
We might need to add:

```yaml
base_images:
  cli-ocm:
    name: cli-ocm
    namespace: ci
    tag: latest
  rosa-aws-cli:
    name: rosa-aws-cli
    namespace: ci
    tag: release
```

To get past some of the CI issues pulling those images:

```
* 2024-05-16T07:32:06Z 261x kubelet: Back-off pulling image "image-registry.openshift-image-registry.svc:5000/ci-op-jcgs5ks6/stable:rosa-aws-cli"
```

I needed to do something similar with the compliance-operator job.

https://github.com/openshift/release/blob/master/ci-operator/config/ComplianceAsCode/compliance-operator/ComplianceAsCode-compliance-operator-master.yaml#L6-L17